src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/BurdenEstimates/BurdenEstimateUploadController.kt

```diff
@@ -26,6 +28,8 @@ class BurdenEstimateUploadController(context: ActionContext,
                                      private val repositories: Repositories,
                                      private val estimatesLogic: BurdenEstimateLogic,
                                      private val estimateRepository: BurdenEstimateRepository,
+                                     private val groupRepository: ModellingGroupRepository,
```

Comment:
These extra repos are no longer needed right? Similarly in the `Upload` controller...

Reply:
Oops, yeah, forgot about that!